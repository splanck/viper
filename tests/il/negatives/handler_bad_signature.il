il 0.1.2
func @main() -> void {
entry:
  eh.push ^bad
  ret
handler ^bad(%err:Error, %token:ResumeTok):
  eh.entry
  resume.next %token
}
