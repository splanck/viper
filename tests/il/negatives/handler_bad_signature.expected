error: main:bad: handler params must be named %err and %tok
